dbt_project.yml

```diff
@@ -0,0 +1,51 @@
+name: 'tuva_claims_demo'
+version: '0.1.0'
+config-version: 2
+
+profile: tuva_claims_demo                 # Configuration required (your profile to connect dbt to your data warehouse)
+
+
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"
+clean-targets:
+  - "target"
+  - "dbt_packages"
+
+vars:
+  # These variables tell the_tuva_project package what
+  # packages to run. If there are some packages you
+  # are not interested in running, this is where you
+  # need to set the relevant variable(s) to 'false':
+  tuva_packages_enabled: true
+  data_profiling_enabled: true
+  claims_preprocessing_enabled: true
+  chronic_conditions_enabled: true
+  readmissions_enabled: true
+  terminology_enabled: true
+
+  # This variable sets the name of the database
+  # where the output of the Tuva Project will be
+  # dumped into. Make sure this database exists
+  # in your data warehouse before you run this package,
+  # since dbt can create schemas in your data warehouse,
+  # but it cannot create databases.
+  tuva_database: tuva_claims_demo_sample
+
+  # This variable allows you to override the sample data seeds and
+  # use the full demo data instead, you may also want to update the
+  # tuva_database variable to not overwrite the sample demo
+  full_data_override: false
```

Comment:
@aneiderhiser I improved this logic a bit. It's now a separate var and not tied to the database name. I also updated the steps in the README to reflect this change.

Reply:
awesome, was going to say after using it I was going to suggest something like this instead.